src/pages/product/card/ProductCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useWishlist } from '../../wishlist/wishlist-context';
+import "../../../App.css";
+const ACTIONS = {
+  ADDTOWISHLIST : "add-to-wishlist", 
+  REMOVEFROMWISHLIST : "remove-from-wishlist"
```

Comment:
Add underscores between words, Instead of writing `ADDTOWISHLIST` write `ADD_TO_WISHLIST`.
You can see that the second one is looking more readable.

Reply:
Done